getdeps.sh

```diff
@@ -62,14 +62,33 @@ function build_cmake() {
     )
 }
 
+function build_gtest_and_gmock() {
+    (
+        mkdir build.test && cd build.test
+        for LIB in gmock gtest
+        do
+          mkdir $LIB && cd $LIB
+          echo "building $LIB..."
+          test -d "/usr/src/$LIB" || die "Missing dir /usr/src/$LIB"
+          cmfile=/usr/src/$LIB/CMakeLists.txt
+          test -e $cmfile || die "Missing cmake file for $LIB"
+          echo ${PWD}
+          echo cmake $LIB $CMAKEFLAGS
+          sudo cmake $cmfile  $CMAKEFLAGS -B./ || die "Cmake failed for $LIB"
```

Comment:
I have to add sudo & -B./ here because it wouldn't work otherwise on my vm.

Reply:
In general, files in /usr/... are not meant for people to write to unless their' updating the system.  The way that I've (tried to) describe involves building everything in a new directory as part of the fboss code checkout, e.g., in "build.test".  If you do it that way, you won't need 'sudo'.  Does that make sense?

Reply:
I see. I should have checked to see if I can remove the sudo after I figured out the -B./ part of it. Thanks! I will update the pr now.